docs/learn/court.md

```diff
@@ -6,8 +6,121 @@ title: Decentralized Court
 Zeitgeist implements a decentralized court to handle disputes that may arise in
 the resolution of prediction markets outcomes.
 
-The court is inspired by [Aragon Court][] and [Kleros][], two projects that have
-already experimented with decentralized court systems.
+### Terminology
 
-[aragon court]: https://example.com
-[kleros]: https://example.com
+- court hopping: joining the court just to receive token emission benefits and then exiting the court
+- court pool: all jurors and delegators
+- draw weight: one slash-able `MinJurorStake` that belongs to one vote weight
+- global dispute: a token voting mechanism for all ZTG holders
+- participant: a juror or delegator account inside the court pool
+- vote weight: the derivative voting value of one draw weight; it represents one `MinJurorStake`
```

Comment:
This needs to be spell-checked. Capitalization at the start of each item, capitalization after the colon and full stop at the end of sentences.